scripts/pre_commit_linter.py

```diff
@@ -129,7 +154,7 @@ def _lint_js_files(node_path, jscs_path, config_jscsrc, input_dir):
     if not files_to_lint:
         print 'There are no JavaScript files to lint. Exiting.'
         print '----------------------------------------'
-        sys.exit(0)
+        return
```

Comment:
If I'm reading it correctly, this returns control to the main `_pre_commit_linter()` function -- which seems reasonable to me given that it's possible that a commit may contain only python files.

If anything, I'd probably remove the "Exiting" part in the comment two lines above; ditto for the similar message in _lint_py_files.